bosh/jobs/cloud_controller_ng/templates/cloud_controller_api.yml.erb

```diff
@@ -168,12 +168,6 @@ logging:
   level: <%= p("cc.logging_level") %>
   max_retries: <%= p("cc.logging_max_retries") %>
 
-<% if_p("metron_endpoint.host", "metron_endpoint.port") do |host, port| %>
-loggregator:
-  router: <%= host %>:<%= port %>
```

Comment:
@wfernandes we need this configuration in Cloud Controller [to talk to Metron (configure the Loggregator Emitter)](https://github.com/cloudfoundry/cloud_controller_ng/blob/master/lib/cloud_controller/runner.rb#L172). I think we only want to remove the url line below.